recipe/meta.yaml

```diff
@@ -1,32 +1,35 @@
 {% set version = "2023.2" %}
+# TODO: Add a note as to where to find this version in the upstream repo
+# Keep this in synch with ...
```

Comment:
@isuruf do you remember how we found the correct version (the first time) I tried to find how but it wasn't too obvious to me how to find it again for the next version.

I wanted to keep this up top so it would be easily identifiable in future updates.

Reply:
https://github.com/KhronosGroup/SPIRV-Tools/blob/main/DEPS#L16